docs/userguide/containers/dockerizing.md

```diff
@@ -24,147 +24,131 @@ Running an application inside a container takes a single command: `docker run`.
 
 ## Run a Hello world
 
-Let's try it now.
+Let's run a hello world container.
 
     $ docker run ubuntu /bin/echo 'Hello world'
     Hello world
 
-And you just launched your first container!
+You just launched your first container!
 
-So what just happened? Let's step through what the `docker run` command
-did.
+In this example:
 
-First we specified the `docker` binary and the command we wanted to
-execute, `run`. The `docker run` combination *runs* containers.
+* `docker run` runs a container.
 
-Next we specified an image: `ubuntu`. This is the source of the container
-we ran. Docker calls this an image. In this case we used the Ubuntu
-operating system image.
+* `ubuntu` is the image you run, for example the Ubuntu operating system image.
+  When you specify an image, Docker looks first for the image on your
+  Docker host. If the image does not exist locally, then the image is pulled from the public
+  image registry [Docker Hub](https://hub.docker.com).
 
-When you specify an image, Docker looks first for the image on your
-Docker host. If it can't find it then it downloads the image from the public
-image registry: [Docker Hub](https://hub.docker.com).
+* `/bin/echo` is the command to run inside the new container.
 
-Next we told Docker what command to run inside our new container:
-
-    /bin/echo 'Hello world'
-
-When our container was launched Docker created a new Ubuntu
-environment and then executed the `/bin/echo` command inside it. We saw
-the result on the command line:
+The container launches. Docker creates a new Ubuntu
+environment and executes the `/bin/echo` command inside it and then prints out:
 
     Hello world
 
-So what happened to our container after that? Well Docker containers
-only run as long as the command you specify is active. Here, as soon as
-`Hello world` was echoed, the container stopped.
+So what happened to the container after that? Well, Docker containers
+only run as long as the command you specify is active. Therefore, in the above example,
+the container stops once the command is executed.
 
-## An interactive container
+## Run an interactive container
 
-Let's try the `docker run` command again, this time specifying a new
-command to run in our container.
+Let's specify a new command to run in the container.
```

Comment:
Looking at this, perhaps we should rephrase this a bit; "a new command in the container" gives the impression that we're reusing the same container, and run a new command in it.